Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<p>Add note that links should be added to the location of these tutorials on the website (once it builds)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>